shared/fs/row.js

```diff
@@ -23,6 +24,8 @@ const mapDispatchToProps = (dispatch: Dispatch) => ({
     if (type === 'folder') {
       dispatch(navigateAppend([{props: {path}, selected: 'folder'}]))
     } else {
+      const localPath = '/tmp/' + Types.getPathName(path)
```

Comment:
This likely won't work on Windows. Why not use os.tmpdir() or somesuch?

Reply:
Good catch! I realized later that in Cecile's design the default location should be in "Downloads", which is what Chrome/Safari downloads into by default. I assume the equivalent thing on Windows is also "Downloads" in user's home folder? Is there an API to get that, or a good way to construct that path from an environment variable?